tests/test_international_days.py

```diff
@@ -0,0 +1,455 @@
+             (26, 6), (5, 8), (6, 7), (7, 6), (31, 5), (19, 3), (20, 4),
+             (8, 2), (9, 3), (24, 11), (14, 2), (25, 10), (26, 1), (3, 1),
+             (27, 12), (30, 11), (19, 8), (17, 8), (20, 3), (21, 2), (8, 9),
+             (22, 9), (9, 4), (23, 12), (10, 3), (11, 10), (14, 5), (26, 8),
+             (3, 6), (27, 5), (1, 10), (4, 1), (28, 6), (5, 4), (29, 7),
+             (16, 4), (6, 11), (17, 1), (20, 10), (18, 6), (21, 11), (23, 5),
+             (10, 10), (11, 3), (14, 12), (12, 4), (15, 9), (13, 9), (1, 3),
+             (4, 8), (2, 8), (6, 2), (7, 11), (18, 1), (23, 2), (12, 3),
+             (13, 2), (24, 6), (1, 4), (25, 7), (2, 3), (26, 4), (27, 9),
+             (6, 5), (30, 6), (31, 7), (16, 8), (19, 5), (20, 6), (21, 7),
+             (8, 4), (9, 1), (10, 6), (25, 8), (15, 5), (3, 3), (30, 9),
+             (4, 4), (28, 9), (31, 12), (29, 12), (19, 10), (8, 11), (9, 10),
+             (10, 1), (11, 12), (14, 11), (15, 2), (3, 8), (27, 7), (1, 8),
+             (4, 3), (5, 2), (29, 5), (16, 6), (6, 9), (17, 7), (7, 12),
+             (18, 4), (21, 9), (22, 6), (23, 7), (10, 8), (11, 5), (12, 6),
+             (15, 11), (13, 7), (24, 1), (1, 1), (4, 10), (2, 6), (5, 11),
+             (7, 5), (24, 8), (25, 5), (2, 1), (26, 2), (27, 11), (30, 4),
+             (28, 12), (7, 2), (31, 1), (16, 10), (19, 7), (17, 11), (21, 5),
+             (8, 6), (22, 10), (9, 7), (10, 4), (11, 9), (14, 6), (15, 7),
+             (3, 5), (4, 6), (28, 11), (5, 7), (29, 10), (16, 1), (19, 12),
+             (17, 12), (18, 11), (9, 8), (23, 8), (14, 9), (12, 9), (15, 12),
+             (13, 12), (3, 10), (28, 2), (29, 3), (17, 5), (18, 2), (22, 4),
+             (23, 1), (11, 7), (13, 5), (24, 3), (1, 7), (25, 2), (2, 4),
+             (5, 9), (6, 6), (30, 3), (7, 7), (8, 1), (24, 10), (25, 11),
+             (30, 10), (31, 3), (16, 12), (19, 9), (17, 9), (20, 2), (21, 3),
+             (8, 8), (22, 8), (9, 5), (10, 2), (11, 11), (14, 4), (12, 12),
+             (25, 12), (15, 1), (26, 11), (3, 7), (27, 2), (1, 11), (28, 5),
+             (31, 8), (5, 5), (29, 8), (16, 3), (6, 10), (17, 2), (20, 9),
+             (18, 9), (21, 12), (22, 3), (23, 10), (12, 11), (13, 10), (3, 12),
+             (1, 12), (2, 11), (29, 1), (7, 8), (23, 3), (12, 2), (13, 3),
+             (24, 5), (1, 5), (2, 2), (26, 7), (6, 4), (30, 1), (7, 1),
+             (19, 2), (20, 5), (8, 3), (9, 2), (24, 12), (14, 3), (25, 9),
+             (30, 8), (28, 8), (19, 11), (20, 12), (18, 12), (21, 1), (8, 10),
+             (9, 11), (14, 10), (15, 3), (26, 9), (3, 9), (27, 4), (1, 9),
+             (4, 2), (28, 7), (31, 10), (5, 3), (29, 6), (16, 5), (6, 8),
+             (20, 11), (18, 7), (21, 10), (22, 1), (9, 12), (23, 4), (10, 11),
+             (11, 2), (12, 5), (15, 8), (13, 8), (1, 2), (4, 9), (2, 9),
+             (5, 12), (6, 3), (7, 10), (13, 1), (24, 7), (25, 6), (26, 5),
+             (27, 8), (30, 7)]
```

Comment:
What about [29/2](https://en.wikipedia.org/wiki/February_29)? :P

Reply:
I have it in the list...